src/cli/domain/package-static-files/minify-file.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var babel = require('babel-core');
+var babelPresetEnv = require('babel-preset-env');
+var CleanCss = require('clean-css');
+var uglifyJs = require('uglify-js');
+
+module.exports = function(fileType, fileContent){
+
+  if(fileType === '.js'){
+
+    var presetOptions = {
+      targets: {
+        browsers: 'ie 8',
+        uglify: true
+      },
+      useBuiltIns: true,
+      modules: false
+    };
+
+    var babelOptions = { presets: [[babelPresetEnv, presetOptions]] },
+        es5 = babel.transform(fileContent, babelOptions).code;
+    
+    return uglifyJs.minify(es5, { fromString: true }).code;
+
+  } else if(fileType === '.css'){
+
+    return new CleanCss().minify(fileContent).styles;
+  }
+
+  return fileContent;
```

Comment:
you shouldn't need to return here as the file extension check is done also at a higher level.